openmc/mgxs/mgxs.py

```diff
@@ -3534,7 +3907,7 @@ def filters(self):
                                endpoint=True)
             filters = [[energy], [energy, energyout, openmc.MuFilter(bins)]]
 
-        return filters
+        return self._add_angle_filters(filters)
```

Comment:
Still not sure I know why.  It saves 10 lines of code from being repeated like 4 times